loadrunner_professional/assets/logs/loadrunner_professional.yaml

```diff
@@ -0,0 +1,71 @@
+id: loadrunner_professional
+metric_id: loadrunner_professional
+facets: null
+pipeline:
+  type: pipeline
+  name: LoadRunner Scenario Information Pipeline
+  enabled: true
+  filter:
+    query: "source:loadrunner "
+  processors:
+    - type: attribute-remapper
+      name: LoadRunner Hostname Tag Creation
+      enabled: true
+      sources:
+        - hostname
+      sourceType: attribute
+      target: lr_hostname
+      targetType: tag
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: LoadRunner Service Tag Creation
+      enabled: true
+      sources:
+        - service
+      sourceType: attribute
+      target: lr_service
+      targetType: tag
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: LoadRunner Version Tag Creation
+      enabled: true
+      sources:
+        - scenario_started.general.version
+        - scenario_ended.general.version
+      sourceType: attribute
+      target: lr_version
+      targetType: tag
+      preserveSource: true
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: LoadRunner Scenario Mode Tag Creation
+      enabled: true
+      sources:
+        - scenario_started.summary.mode
+      sourceType: attribute
+      target: lr_scenario_mode
```

Comment:
If the idea is to identify this attribute as specific to load runner. Can we namespace with `loadrunner` instead of `lr` that is both less readable for users and can also describe other technologies.